src/main/java/org/omnifaces/util/JNDIObjectLocator.java

```diff
@@ -13,28 +13,33 @@
 
 package org.omnifaces.util;
 
+import java.beans.ConstructorProperties;
 import static java.util.Collections.emptyMap;
-import static java.util.Objects.requireNonNull;
 import static org.omnifaces.util.Exceptions.is;
-import static org.omnifaces.util.JNDI.JNDI_NAMESPACE_APPLICATION;
-import static org.omnifaces.util.JNDI.JNDI_NAMESPACE_GLOBAL;
-import static org.omnifaces.util.JNDI.JNDI_NAMESPACE_MODULE;
 import static org.omnifaces.util.JNDI.JNDI_NAME_PREFIX_ENV_ENTRY;
 import static org.omnifaces.util.JNDI.guessJNDIName;
 import static org.omnifaces.util.Reflection.toClassOrNull;
-import static org.omnifaces.util.Utils.coalesce;
 
 import java.io.Serializable;
 import java.lang.annotation.Annotation;
+import java.util.ArrayList;
+import java.util.Collections;
+import static java.util.Collections.unmodifiableMap;
 import java.util.Hashtable;
+import java.util.LinkedHashMap;
 import java.util.Map;
+import java.util.Map.Entry;
+import static java.util.Objects.requireNonNull;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 
 import javax.naming.InitialContext;
 import javax.naming.NameNotFoundException;
 import javax.naming.NamingException;
+import static org.omnifaces.util.JNDI.JNDI_NAMESPACE_APPLICATION;
+import static org.omnifaces.util.JNDI.JNDI_NAMESPACE_GLOBAL;
+import static org.omnifaces.util.JNDI.JNDI_NAMESPACE_PREFIX;
```

Comment:
Ordering of imports not anymore consistent with rest of library.

Reply:
fixed